src/javascript/app/pages/user/metatrader/metatrader.ui.js

```diff
@@ -722,11 +725,11 @@ const MetaTraderUI = (() => {
         const selected_type = $form && $form.find('.step-2 .selected').attr('data-acc-type');
         // if user selected account type using the form, send that
         if (selected_type) {
-            return `${$form.find('.step-1 .selected').attr('data-acc-type') || 'real'}_${$form.find('.step-2 .selected').attr('data-acc-type')}`;
+            return MetaTraderConfig.getCleanAccType(`${$form.find('.step-1 .selected').attr('data-acc-type') || 'real'}_${$form.find('.step-2 .selected').attr('data-acc-type')}`);
         }
         // otherwise they are adding more server to their current account type
         const saved_mt5_account = Client.get('mt5_account');
-        return MetaTraderConfig.getCleanAccType(saved_mt5_account);
+        return saved_mt5_account;
```

Comment:
Can simplify with just :
```suggestion
        return Client.get('mt5_account');
```

Reply:
Yes, nice catch 👍🏻